toolkits/springer/packages/springer-listing/README.md

```diff
@@ -6,72 +6,100 @@ Optional image shrinks besides the title and text as the viewport becomes smalle
 
 ## Usage
 
-```html
-<!-- example of listing with text and image -->
-<ul>
-    <li>
-        <article class="c-listing c-listing--row">
-            <div class="c-listing__text-column">
-                <h4 class="c-listing__title">
-                    Title of an article
-                </h4>
-                <div class="c-listing__text">
-                    Some introduction text for article
-                </div>
-            </div>
-            <!-- optional image -->
-            <div class="c-listing__image">
-                <img src="article-image.png" alt="">
-            </div>
-        </article>
-    </li>
-</ul>
-
-<!-- example of listing in a search result -->
-<ul>
-    <li>
-        <article class="c-listing">
-            <div class="c-meta">
-                <p class="c-meta__item">Book Review</p>
-            </div>
-        
-            <div class="u-margin-bottom-sm">
-                <h3 class="c-listing__title">
-                    <a href="#">Title</a>    
-                </h3>
-                <p class="c-listing__authors">Author 1, Author 2, Author 3</p>
-            </div>
-        
-            <div class="u-margin-bottom-sm">
-                <div class="c-listing__text c-listing__text--light">
-                    <p>
-                        <em>Genomics, Society and Policy</em>
-                        2012
-                        <span>8</span>:38
-                    </p>
-                    <p>Published on: <span>15 September 2012</span></p>
-                </div>
-            </div>
-        
-            <ul class="c-listing__view-options">
-                <li>
-                    <a href="#">
-                        <svg class="c-listing__view-options-icon" width="16" height="16" aria-hidden="true">
-                            <use xlink:href="#icon-chevron-right"></use>
-                        </svg>
-                        <span>Full Text</span>
-                    </a>
-                </li>
-                <li>
-                    <a href="#">
-                        <svg class="c-listing__view-options-icon" width="16" height="16" aria-hidden="true">
-                            <use xlink:href="#icon-chevron-right"></use>
-                        </svg>
-                        <span>PDF</span>
-                    </a>
-                </li>
-            </ul>                
-        </article>
-    </li>
-</ul>
+# Usage
+
+Import the Sass:
+
+```scss
+@import '../node_modules/@springernature/springer-listing/scss/10-settings/listing';
```

Comment:
Should the filename now be `listings` instead?

Reply:
@benjclark Oh wait you just mean for the scss file. Yeah. FWIW, I'm not totally comfortable this is listing->listings but I saw no other way to encapsulate it as a list, which should be enforced for accessibility and the group styling.

Reply:
![image](https://user-images.githubusercontent.com/5796370/148238374-9d9c44a3-0874-4d9e-b8ca-a3ff23144cca.png)

Maybe it's ok for this component to stay as `c-listing` ? As even in its singular form (listing) its meaning is an entire list or catalogue or something.

Reply:
@benjclark You genius, I hadn't thought of that! I just reverted it all to singular.